packages/gasket-data-react/src/withGasketData.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { GasketDataProvider } from './GasketDataProvider';
+
+let gasketData: Record<string, unknown> = null;
```

Comment:
A shared global like this won't work since the node server could be servicing multiple concurrent requests; this could cause customer data to bleed across sessions.

`gasketData` should be returned in the `getInitialProps` call and then injected into the context, not pulled from the closure.